test/e2e/testing-manifests/storage-csi/hostpath/usage/csi-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
do we need manifests for test pvc and pods?

Reply:
That could be removed. I kept it for the sake of completeness ("this directory is a copy of..."), but the tests themselves don't need them.

Reply:
Done.